README.md

```diff
@@ -18,4 +18,11 @@ in stdenv.mkDerivation {
 
 # Usage
 
-tcrdd [-g|-r]
+```
+tcrdd [options...] testCommand [arguments...]
+
+Options:
+    -g                  assume the tests will pass (green)
+    -r                  assume the tests will fail (red)
+    -m MESSAGE          use the provided commit message
+```
```

Comment:
Why adding usage in the README if it's in the code ?
What about saying something like : To learn how to use the command, type `tcrdd -h` ?
My goal being avoiding to write the same thing twice (and maintain both :) )

Reply:
Fair enough. I left the basic command structure, removed the option details and added the bit you requested stating that one should ask the command for help. Is this ok ?